src/middleware/validateApiKey.js

```diff
@@ -64,28 +24,16 @@ export default async (req, res, next) => {
     if (!apiKey && isDevelopment) {
       apiKey = FALLBACK_API_KEY;
     }
-    if (!apiKey) {
-      throw new Error('X-API-Key Header doesn\'t exist');
-    }
```

Comment:
these lines should be left alone

Reply:
@davydocsurg bumping this message

Reply:
@davydocsurg bumping this message again - can you respond mentioning that you've addressed this?